diff -up pigz-2.5/Makefile.1~ pigz-2.5/Makefile
--- pigz-2.5/Makefile.1~	2021-01-24 06:19:13.000000000 +0100
+++ pigz-2.5/Makefile	2021-01-25 03:19:15.954026445 +0100
@@ -1,6 +1,6 @@
-CC=gcc
-CFLAGS=-O3 -Wall -Wextra -Wno-unknown-pragmas -Wcast-qual
-LDFLAGS=
+CC?=gcc
+CFLAGS+=-O3 -Wall -Wextra -Wno-unknown-pragmas -Wcast-qual
+LDFLAGS+=
 # CFLAGS=-O3 -Wall -Wextra -Wno-unknown-pragmas -Wcast-qual -g -fsanitize=thread
 # LDFLAGS=-g -fsanitize=thread
 # CFLAGS=-O3 -Wall -Wextra -Wno-unknown-pragmas -Wcast-qual -g -fsanitize=address
@@ -12,7 +12,7 @@ ZOP=deflate.o blocksplitter.o tree.o lz7
 # use gcc and gmake on Solaris
 
 pigz: pigz.o yarn.o try.o $(ZOP)
-	$(CC) $(LDFLAGS) -o pigz pigz.o yarn.o try.o $(ZOP) $(LIBS)
+	$(CC) $(LDFLAGS) $(CFLAGS) -o pigz pigz.o yarn.o try.o $(ZOP) $(LIBS)
 	ln -f pigz unpigz
 
 pigz.o: pigz.c yarn.h try.h $(ZOPFLI)deflate.h $(ZOPFLI)util.h
